src/lib/components/Navbar.svelte

```diff
@@ -89,86 +89,97 @@ The navbar component.
+                                props: { options: Object.keys(chapters) }
+                            },
+                            Verse: {
+                                component: SelectGrid,
+                                props: { options: Object.keys(chapters[nextRef.chapter]) }
+                            }
+                        }}
+                        active="Book"
+                        on:menuaction={navigateReference}
+                    />
+                </svelte:fragment>
+            </Dropdown>
+        {/if}
         <!-- Chapter Selector -->
-        <Dropdown>
-            <svelte:fragment slot="label">
-                {nextRef.chapter}
-                <DropdownIcon />
-            </svelte:fragment>
-            <svelte:fragment slot="content">
-                <TabsMenu
-                    bind:this={chapterSelector}
-                    options={{
-                        Chapter: {
-                            component: SelectGrid,
-                            props: { options: Object.keys(chapters) }
-                        },
-                        Verse: {
-                            component: SelectGrid,
-                            props: { options: Object.keys(chapters[nextRef.chapter]) }
-                        }
-                    }}
-                    active="Chapter"
-                    on:menuaction={navigateReference}
-                />
-            </svelte:fragment>
-        </Dropdown>
+        {#if $globalConfig.mainFeatures["show-chapter-numbers"]}
+            <Dropdown>
+                <svelte:fragment slot="label">
+                    {nextRef.chapter}
+                    <DropdownIcon />
+                </svelte:fragment>
+                <svelte:fragment slot="content">
+                    <TabsMenu
+                        bind:this={chapterSelector}
+                        options={{
+                            Chapter: {
+                                component: SelectGrid,
+                                props: { options: Object.keys(chapters) }
+                            },
+                            Verse: {
+                                component: SelectGrid,
+                                props: { options: Object.keys(chapters[nextRef.chapter]) }
+                            }
+                        }}
+                        active="Chapter"
+                        on:menuaction={navigateReference}
+                    />
+                </svelte:fragment>
+            </Dropdown>
+        {/if}
     </div>
     <div class="dy-navbar-end fill-base-content">
         <!-- Mute/Volume Button -->
-        <button class="dy-btn dy-btn-ghost dy-btn-circle">
-            <label class="dy-swap">
-                <!-- this hidden checkbox controls the state -->
-                <input type="checkbox" bind:checked={$audioActive} />
+        {#if $globalConfig.mainFeatures["audio-turn-on-at-startup"]} 
```

Comment:
The visibility of the audio button is the nav bar is based on whether there is audio in the current chapter.  The "audio-turn-on-at-startup" controls whether the audio toolbar at the bottom displays when the app is launched (if the current chapter has audio).  If this is unchecked, then the audio toolbar is not displayed if the current chapter has audio.